### DISCUSSION TITLE
I just got banned from discord for suggesting not putting malware on the docs

### DISCUSSION THREAD

**Discussion by davidromrell** [FabricMC/fabric-api#3765] [FAQ] [answered]
is this secretly a terrorist organization

**Comment by Shnupbups** [comment, marked as answer]
You were banned for continuously bringing up terrorism, and giving unsubstantiated claims that VirusTotal was backed by identity theft organisations and is involved in terrorism. You made an allusion to 9/11 and were warned. After further warning, and no proof given to your claims, you were given a 12 week ban.
By making this discussion post, you have proven you are obviously unwilling to listen and learn, or discuss in good faith.

**Reply by davidromrell** [reply]
You stated in unrepeatable words due to not being able to access them saying there was no such thing as terrorists. I got banned for saying they exist. Note: prior to chatting I sent a detailed report to the FBI.
